src/main/java/org/fundacionjala/pivotal/api/APIRequestManager.java

```diff
@@ -0,0 +1,66 @@
+package org.fundacionjala.pivotal.api;
+
+import com.jayway.restassured.response.Response;
+import com.jayway.restassured.specification.RequestSpecification;
+
+import java.util.Map;
+import static com.jayway.restassured.RestAssured.given;
+
+/**
+ * APIRequestManager.java
+ * Class that implements the methods to get, post, delete and put through Pivotal API.
+ */
+public final class APIRequestManager {
+    /**
+     * Constructor.
+     */
+    private APIRequestManager() {
+    }
+
+    /**
+     * Response attribute to get request.
+     */
+    private static Response response;
+    private static RequestSpecification requestSpecificacion;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "requestSpecificacion" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)